.eslintignore

```diff
@@ -0,0 +1,5 @@
+/.nyc_output/
+/lib/
+/node_modules/
+!.mocharc.js
```

Comment:
Is this needed? Are these files ignored by default?

Reply:
Yes they are ignored by default